pages/details/[id].js

```diff
@@ -0,0 +1,42 @@
+import {
+  getItemByDatabaseId,
+  getItemByDatabaseIdv2,
+} from "../../components/Db";
+
+export function getServerSideProps(context) {
```

Comment:
Not sure but I think this should be an async function

Reply:
Here I already had Felix look over it and I think it doesn't have to be async since the data is pulled from db.js.